docs/release-notes/cloud/cloud-2024-04.md

```diff
@@ -0,0 +1,98 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2024.
+included_jira_versions: ['2024.04']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.272
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.305
+---
+
+# Cloud April 2024
+
+These release notes are for the Codacy Cloud updates during April 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+-->
+
+## Product enhancements
+
+-   Improved [Security and risk management](../../organizations/managing-security-and-risk.md) with:
+    -   A redesigned overview page, offering comprehensive metrics and summary graphs for better visibility and tracking of security findings.
+    -   An improved findings table, which now includes a streamlined view of security findings. (TAROT-2546)
+
+    !!! note
+        With this update, all references to "security items" will be updated to "security findings" to align with the new features.
```

Comment:
@nicklem Tweak to avoid "update/updated" repetition. Let me know if it's ok.

```suggestion
        To align with the new features, all references to "security items" were updated to "security findings".
```